tests/performance/random_fixed_string.xml

```diff
@@ -0,0 +1,9 @@
+<test>
```

Comment:
Please also provide functional test for correctness.

Reply:
Done.